### DISCUSSION TITLE
Is this project still relevant?

### DISCUSSION THREAD

**Discussion by abacao** [abacao/Boot2Kodi#17] [Announcements]
Should I update the project to the new Ubuntu 22.04?

**Comment by aleksandr-a-li** [comment]
Yes, it's still relevant, please update it to 22.04
I tried installing this on Ubuntu Server 22.04 myself and it works great.
Thank you!

Added: In my case Ubuntu Server 22.04 already has Git after installation, the "sudo apt install git -y" command was not needed.

**Comment by mrobe** [comment]
This worked well for me with Ubuntu 24.04.1 and Kodi 20.2 — thanks !
